<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="color_format" value="yuv422p" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  
  <node name="camera_convert" pkg="t1_robot" type="node_1.py" output="screen">
  </node>
  
  <node name="rec_node_1" pkg="t1_robot" type="toDiscord.py" output="screen" 
  launch-prefix="xterm -e">
  </node>

  <node name="ROH_image_view" pkg="image_view" type="image_view" respawn="false" output="screen" >
    <remap from="image" to="/topic_DisplayROH"/>
    <param name="autosize" value="true" />
    <param name="_image_transport" value="image_raw" />
  </node>
</launch>
